library/src/main/java/kwasm/format/text/Type.kt

```diff
@@ -18,6 +18,7 @@ import kwasm.ast.Limit
 import kwasm.format.ParseContext
 import kwasm.format.ParseException
 import kwasm.format.shiftColumnBy
+import kwasm.format.text.token.IntegerLiteral
```

Comment:
No changes refrence this import, is it needed?

Reply:
Similar to `Name` - this is because IntegerLiteral was moved